flake.nix

```diff
@@ -218,14 +214,14 @@
           projectDir ? self, # TODO: only include relevant files/folders
```

Comment:
let's change the signature here to start with `{ self, system, ...` and lets also move `jupyterlab.nix`. The idea is to have it look similar as kernels.

Reply:
to include poetry2nix we should still use overlay, but only in the scope of jupyterlab.

```
{
  self,
  system,
  poetry2nix ? self.inputs.poetry2nix,
  pkgs ? import self.inputs.nixpkgs { inherit system; overlays = [ poetry2nix.overlay ]; },
  ...
}:
...
```

Reply:
done